{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -26,3 +26,6 @@ compose-down:
 
 compose-build:
 	cd docker/ && docker compose build
+
+api-migrate:
+	cd docker/ && docker compose run --rm api make migrate
```

Comment:
I think we should move this up after `api-lock` or even after `api-check-migrations` to keep some kind of order here